crates/iota-data-ingestion-core/src/executor.rs

```diff
@@ -105,7 +107,12 @@ impl<P: ProgressStore> IndexerExecutor<P> {
                             self.progress_store.save(task_name.clone(), sequence_number).await?;
                             let seq_number = self.progress_store.min_watermark()?;
                             if seq_number > reader_checkpoint_number {
-                                gc_sender.send(seq_number).await?;
+                                gc_sender.send(seq_number).await.map_err(|_| {
+                                    IngestionError::Channel(
+                                        "unable to send GC operation to checkpoint reader, receiver half dropped"
```

Comment:
Nit: `closed` is more precise
```suggestion
                                        "unable to send GC operation to checkpoint reader, receiver half closed"
```